<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="1" name="pytest" skips="0" tests="8" time="0.447"><testcase classname="pytests.test_index" file="pytests/test_index.py" line="10" name="test_conda_environment_activated" time="0.0022084712982177734"></testcase><testcase classname="pytests.test_index" file="pytests/test_index.py" line="13" name="test_restaurant_name_func" time="0.0006818771362304688"></testcase><testcase classname="pytests.test_index" file="pytests/test_index.py" line="18" name="test_restaurant_rating_func" time="0.0006508827209472656"></testcase><testcase classname="pytests.test_index" file="pytests/test_index.py" line="23" name="test_is_better_func" time="0.0020101070404052734"></testcase><testcase classname="pytests.test_index" file="pytests/test_index.py" line="29" name="test_is_cheaper_func" time="0.002641916275024414"><failure message="AssertionError: assert False == True
 +  where False = is_cheaper({&apos;categories&apos;: [{&apos;alias&apos;: &apos;mexican&apos;, &apos;title&apos;: &apos;Mexican&apos;}, {&apos;alias&apos;: &apos;diners&apos;, &apos;title&apos;: &apos;Diners&apos;}, {&apos;alias&apos;: &apos;tradameri...0808088832532, &apos;longitude&apos;: -106.619402244687}, &apos;display_phone&apos;: &apos;(505) 266-0550&apos;, &apos;distance&apos;: 4033.6583235266075, ...}, {&apos;categories&apos;: [{&apos;alias&apos;: &apos;burgers&apos;, &apos;title&apos;: &apos;Burgers&apos;}, {&apos;alias&apos;: &apos;sandwiches&apos;, &apos;title&apos;: &apos;Sandwiches&apos;}, {&apos;alias&apos;: &apos;s...{&apos;latitude&apos;: 35.10871, &apos;longitude&apos;: -106.56739}, &apos;display_phone&apos;: &apos;(505) 881-5293&apos;, &apos;distance&apos;: 3571.724649307866, ...})">def test_is_cheaper_func():
        assert &apos;is_cheaper&apos; in ipynb.fs.full.index.__dict__, &quot;The is_cheaper function must be defined&quot;
&gt;       assert is_cheaper(frontier_restaurant, fork_fig) == True
E       AssertionError: assert False == True
E        +  where False = is_cheaper({&apos;categories&apos;: [{&apos;alias&apos;: &apos;mexican&apos;, &apos;title&apos;: &apos;Mexican&apos;}, {&apos;alias&apos;: &apos;diners&apos;, &apos;title&apos;: &apos;Diners&apos;}, {&apos;alias&apos;: &apos;tradameri...0808088832532, &apos;longitude&apos;: -106.619402244687}, &apos;display_phone&apos;: &apos;(505) 266-0550&apos;, &apos;distance&apos;: 4033.6583235266075, ...}, {&apos;categories&apos;: [{&apos;alias&apos;: &apos;burgers&apos;, &apos;title&apos;: &apos;Burgers&apos;}, {&apos;alias&apos;: &apos;sandwiches&apos;, &apos;title&apos;: &apos;Sandwiches&apos;}, {&apos;alias&apos;: &apos;s...{&apos;latitude&apos;: 35.10871, &apos;longitude&apos;: -106.56739}, &apos;display_phone&apos;: &apos;(505) 881-5293&apos;, &apos;distance&apos;: 3571.724649307866, ...})

pytests/test_index.py:32: AssertionError</failure></testcase><testcase classname="pytests.test_index" file="pytests/test_index.py" line="35" name="test_high_rating_func" time="0.0006482601165771484"></testcase><testcase classname="pytests.test_index" file="pytests/test_index.py" line="41" name="test_mean_review_count_func" time="0.0005917549133300781"></testcase><testcase classname="pytests.test_index" file="pytests/test_index.py" line="45" name="test_mediam_review_count_func" time="0.0006902217864990234"></testcase></testsuite>